### PR TITLE
Feature remove pedigree 200313

### PR DESCRIPTION
**Pull request by mikaell**
This PR adds a functionality or fixes a bug.

**How to test**:
1. how to install it
Check out and install Scout as usual.

- load one cancer config
`scout --demo load case demo/cancer.load_config.yaml  -u`

- load one ordinary config
`scout --demo load case demo/643594.config.yaml  -u`

1. how to test it, possibly with real cases/data

With two loaded cases, examine and compare them. Note that no Pedigree panel is displayed in the cancer case.

**Expected outcome**:

The functionality should be working

Screenshot 2020-03-13 at 11 50 03

Screenshot 2020-03-13 at 11 50 36

**Review:**
- [x] code approved by @hassanfa 
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/cases/templates/cases/case.html [hunk 1]
```diff
@@ -76,19 +76,22 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
...
         <div class="col-md-4">
           {% if case.madeline_info and case.individuals|length > 1 %}
             {{ pedigree_panel(case) }}
           {% else %}
             <p>No pedigree picture available.</p>
           {% endif %}
         </div>
+        {% endif %}
 
-        <div class="col-md-8">
```
Comment (mikaell) [inline scout/server/blueprints/cases/templates/cases/case.html]: Replacing `col-md-8` with `col` to make the lines expand the whole column. Are we using the other screen layouts? (small, medium, large, XL)

Comment (hassanfa) [reply scout/server/blueprints/cases/templates/cases/case.html]: I don't know about this. Maybe @northwestwitch can chime in?

Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/case.html]: Agreed - @northwestwitch is the guru there! One way to think about it is that the sizes refer to the smallest size intended. And scout is generally not designed for the very small devices.

Some docs here https://getbootstrap.com/docs/4.4/layout/grid and more generally here https://getbootstrap.com/docs/4.4/layout/overview/#responsive.

Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/case.html]: I'll take a look as soon as I finish reviewing another PR!

Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/case.html]: Haha definitely not the guru, I just google all the time! 😄 
We generally use the layouts mentioned by @dnil. It works already with the code that you wrote, but if you want to be super sure you could write col-12 to force the column to stretch over the entire line!

File: scout/server/blueprints/cases/templates/cases/case.html [hunk 2]
```diff
@@ -76,19 +76,22 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
...
+        <div class="col">
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/case.html]: You could use a conditional here, like col-12 if it's cancer otherwise col-8

Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/case.html]:
```suggestion
        <div class="{% if case.track != 'cancer' %}col-8{%else%}col-12{%endif%}"">
```

**Comment by northwestwitch** [comment]
I take it as a yes!

**Pull request merged**